packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2876,7 +2878,8 @@ export const EndCallButton: (props: EndCallButtonProps) => JSX.Element;
 
 // @public
 export interface EndCallButtonProps extends ControlBarButtonProps {
-    onHangUp?: () => Promise<void>;
+    enableEndCallMenu?: boolean;
+    onHangUp?: (forEveryone?: boolean) => Promise<void>;
```

Comment:
Just a question, is the implementation of `onHangUp` completely controlled by Contoso? Is there specific role that can perform this action or any specific participant?

Reply:
the onHangUp in our case is provided by sdk (usePropsFor)
But if Contoso wants to provide their own function, that is possible too

For role, we have some rules to determine whether the menu button would be enabled:

1. Whether developer turn on the flag in composite option
2. Whether capabilities don't say no to "HangUpToEveryone"
3. Currently it is only enabled in desktop, so we disable it in mobile